Jenkinsfile

```diff
@@ -72,26 +72,6 @@ pipeline {
         '''
       }
     }
-    stage('Integration: KEVM') {
```

Comment:
Please re-enable these tests, but otherwise this is fine with me!

Reply:
sure. I just want to measure how much overhead adding debug flag would incur.